classes/Tools.php

```diff
@@ -4127,9 +4127,10 @@ public static function formatBytes($size, $precision = 2)
             return '0';
         }
         $base = log($size) / log(1024);
+        $baseUnity = Context::getContext()->getTranslator()->trans('Bytes', [], 'Shop.Theme.Catalog');
         $suffixes = ['', 'k', 'M', 'G', 'T'];
```

Comment:
```suggestion
        $suffixes = ['o', 'ko', 'Mo', 'Go', 'To'];
```
Should be enough

Reply:
@PierreRambaud  why ?

Reply:
Because it's not needed to add a translation for a unit.
kBytes is not the right way to write it :sweat_smile: it's `KByte(s)`, imho the short usage is the common one. People are often talking about ko/Mo than KBytes/MBytes

Reply:
But `Mo = Mega octets`, its french right ?

Reply:
afaik no, https://en.wikipedia.org/wiki/Octet_(computing) but it's more used in France, or Canada because byte is ambiguous (1 byte/1 octet = 8 bits) :sweat_smile: 


```suggestion
        $suffixes = ['Byte(s)', 'KByte(s)', 'MBytes(s)', 'GByte(s)', 'TByte(s)'];
```
wdyt about this one?

Reply:
@PierreRambaud  That's fine with me, but I think it should still be translatable.

Reply:
I update my second command to be sure it's correctly translated

Reply:
Done ! Thanks @PierreRambaud